### DISCUSSION TITLE
What do you like about Museeks?

### DISCUSSION THREAD

**Discussion by martpie** [martpie/museeks#644] [General]
Hello 👋  

while working on the re-design of Museeks, I am interested in your opinions on what you like about Museeks? It would be extremely useful to understand better what UX paradigms I should keep, improve, or throw away.

Examples: 

- UX
- "compact-ness"
- Small focus
- Simplicity
- etc

If there are things you don't like, feel free to share them, but they'll be less important.

**Comment by blackshibe** [comment]
It plays music and looks pleasing, unlike a surprisingly long list of alternatives. There's no extra information so the window can stay small.

**Comment by ilyasbabu** [comment]
Does it have musics online? If yes does it have every musics?

**Comment by lsooxlla8** [comment]
Ability to just add folders from any place in your computer and shuffle across whole library
